vmdb/spec/models/dialog_field_radio_button_spec.rb

```diff
@@ -110,7 +110,7 @@
         context "when the dialog field is dynamic" do
           before do
             dialog_field_radio_button.dynamic = true
-            dialog_field_radio_button.default_value = [["test", 321]]
+            dialog_field_radio_button.default_value = "test321"
```

Comment:
The default_value here should not have been an array in the first place.  The values of the radio button would be an array of arrays like this: ```[[value1, display_name1], [value2, display_name2]]```.  The default value would be the first element of one of the inner arrays.

I would expect this line to read: ```dialog_field_radio_button.default_value = "test"```.  Then the test below would expect ```@value``` to be just "test".